dcc-lsystem/src/renderer.rs

```diff
@@ -1,17 +1,27 @@
 use std::collections::HashMap;
 
+#[cfg(feature = "image_renderer")]
```

Comment:
I haven't fully thought it through, but it might be work making a new module containing all of the image_renderer specific code.  I _think_ backwards compat can be maintained by doing a re-export?  e.g.:
```rust
// in `lib.rs`
#[cfg(feature = "image_renderer")]
pub mod whatever;

// in `renderer.rs:
#[cfg(feature = "image_renderer")]
pub use crate::whatever::ImageRendererOptions
// etc
```

Reply:
Yes, that works without needing to change any of the examples. Good idea.

It looks so much better without all those `#[cfg(feature = "image_renderer")]` everywhere.
Had to add two `pub (crate)` clauses on two (previously private) renderer fields.